src/sgame/sg_bot_skilltree.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+===========================================================================
+
+Unvanquished GPL Source Code
+Copyright (C) 2022 Antoine Fontaine
```

Comment:
We don't usually write any specific people's name in the license header

Reply:
So what should we write?

Reply:
"Unvanquished Developers" apparently

Reply:
Edited